src/frontend/src/pages/myfeeds.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Container, Box, Typography, TextField, Grid, Card, IconButton } from '@material-ui/core';
+import { AccountCircle, RssFeed, HelpOutline, Add } from '@material-ui/icons';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
+  margin: {
+    margin: theme.spacing(2),
+  },
+  button: {
+    padding: '3px 0 3px 0',
+  },
+}));
+
+export default function MyFeeds() {
+  const classes = useStyles();
+  return (
+    <div className={classes.margin}>
+      <Container maxWidth="xs" bgcolor="aliceblue">
+        <Card>
+          <Box px={2} py={1}>
+            <Typography variant="h3" component="h3" align="center">
+              My Feeds
+            </Typography>
+            <Grid container spacing={5}>
+              <Grid item>
+                <Grid container spacing={1} alignItems="flex-end">
+                  <Grid item>
+                    <AccountCircle />
+                  </Grid>
+                  <Grid item>
+                    <TextField disabled label="John Doe" />
```

Comment:
Can we add the id = "author"

Reply:
Done: https://github.com/Seneca-CDOT/telescope/pull/735/commits/24c6a44423e887991d032f2425cb39a0ab044943